File: backend/api/src/main/scala/com/github/ai/leetcodequiz/apisc/response/GetQuestionnairesResponse.scala
package com.github.ai.leetcodequiz.apisc.response

import com.github.ai.leetcodequiz.apisc.QuestionnaireItemDto
import zio.json.{JsonDecoder, JsonEncoder}

case class GetQuestionnairesResponse(
  questionnaires: List[QuestionnaireItemDto]
) derives JsonEncoder, JsonDecoder

File: android/backend-api/src/main/kotlin/com/github/ai/leetcodequiz/api/response/GetQuestionnairesResponse.kt
package com.github.ai.leetcodequiz.api.response

import kotlinx.serialization.Serializable
import com.github.ai.leetcodequiz.api.QuestionnairesItemDto

@Serializable
data class GetQuestionnairesResponse(
    val questionnaires: List<QuestionnairesItemDto>
)